external/storm-jms/src/main/java/org/apache/storm/jms/spout/JmsMessageID.java

```diff
@@ -14,20 +14,21 @@
 
 import java.io.Serializable;
 
+@SuppressWarnings("checkstyle:AbbreviationAsWordInName")
 public class JmsMessageID implements Comparable<JmsMessageID>, Serializable {
 
-    private String jmsID;
+    private String jmsId;
 
     private Long sequence;
 
-    public JmsMessageID(long sequence, String jmsID) {
-        this.jmsID = jmsID;
+    public JmsMessageID(long sequence, String jmsId) {
+        this.jmsId = jmsId;
         this.sequence = sequence;
     }
 
-
-    public String getJmsID() {
-        return this.jmsID;
+    @SuppressWarnings("checkstyle:AbbreviationAsWordInName")
+    public String getJmsId() {
```

Comment:
I don't think we should rename this, unless users can't directly access this class?